CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 ### Bug
 
+- Fix hardware wallet scroll bug[#890](https://github.com/MyEtherWallet/MyEtherWallet/pull/890)
```

Comment:
Place it at the top of the file since this will be in a different version:
```
   ### Bug
   
    - Fix hardware wallet scroll bug[#890](https://github.com/MyEtherWallet/MyEtherWallet/pull/890)
```